PowerEditor/src/WinControls/StatusBar/StatusBar.cpp

```diff
@@ -45,6 +47,84 @@ void StatusBar::init(HINSTANCE /*hInst*/, HWND /*hPere*/)
 	assert(false and "should never be called");
 }
 
+LRESULT CALLBACK StatusBarSubclass(
+	HWND hWnd,
+	UINT uMsg,
+	WPARAM wParam,
+	LPARAM lParam,
+	UINT_PTR uIdSubclass,
+	DWORD_PTR dwRefData
+)
+{
+	UNREFERENCED_PARAMETER(dwRefData);
+	UNREFERENCED_PARAMETER(uIdSubclass);
+
+	switch (uMsg) {
+	case WM_ERASEBKGND:
+	{
+		if (!NppDarkMode::IsEnabled()) {
+			return DefSubclassProc(hWnd, uMsg, wParam, lParam);
+		}
+
+		RECT rc;
+		GetClientRect(hWnd, &rc);
+		FillRect((HDC)wParam, &rc, NppDarkMode::GetBackgroundBrush());
+		return 1;

```

Comment:
Why 1? I suppose, this should return TRUE (here and in other message procedures).